lib/active_merchant/billing/gateways/pay_gate_xml.rb

```diff
@@ -251,6 +270,13 @@ def commit(action, request)
           :authorization  => response[:tid]
         )
       end
+      def commit_capture(action, authorization, request)
```

Comment:
This method is largely a duplication of `#commit`. If you want to add the ability to pass in authorizations, augment the `#commit` method rather than duplicating it. See other integrations for examples of this.